core/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Andrew Jones <ascjones@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-frame-metadata = { package = "frame-metadata", git = "https://github.com/paritytech/frame-metadata", branch = "aj-substrate", default-features = false, features = ["v13"] }
+frame-metadata = { package = "frame-metadata", git = "https://github.com/montekki/frame-metadata", branch = "fs-substrate", default-features = false, features = ["v14"] }
```

Comment:
Would patching `scale-info` work here instead to get the latest `scale-info`?

```
[patch.crates-io]
scale-info = { git = "https://github.com/paritytech/scale-info", branch = "aj-substrate" }
```

Reply:
I don't think so it will not affect the dependency of `frame-metadata` on `scale-info` so there would still be two versions then I think

Reply:
Can you give it a try to find out? I thought `patch` replaced the resolution of transient dependencies, too.

Reply:
Yes that worked

Reply:
Thanks. Better than depending on a fork.